201-eventhubs-create-namespace-and-enable-archive/README.md

```diff
@@ -0,0 +1,7 @@
+# 201-eventhubs-create-namespace-and-enable-archive
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F201-event-hubs-create-event-hub-and-enable-archive%2Fazuredeploy.json" target="_blank">
```

Comment:
this link is incorrect; it should be 201-eventhubs-create-namespace-and-endable-archive